sys-devel/llvm/llvm18-18.1.1.recipe

```diff
@@ -0,0 +1,788 @@
+	devel:libclangserialization$secondaryArchSuffix = $portVersion
+	devel:libclangstaticanalyzercheckers$secondaryArchSuffix = $portVersion
+	devel:libclangstaticanalyzercore$secondaryArchSuffix = $portVersion
+	devel:libclangstaticanalyzerfrontend$secondaryArchSuffix = $portVersion
+	devel:libclangsupport$secondaryArchSuffix = $portVersion
+	devel:libclangtidy$secondaryArchSuffix = $portVersion
+	devel:libclangtidyabseilmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyalteramodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyandroidmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyboostmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidybugpronemodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidycertmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyconcurrencymodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidycppcoreguidelinesmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidydarwinmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyfuchsiamodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidygooglemodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyhicppmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidylinuxkernelmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyllvmlibcmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyllvmmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidymain$secondaryArchSuffix = $portVersion
+	devel:libclangtidymiscmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidymodernizemodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidympimodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyobjcmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyopenmpmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyperformancemodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyplugin$secondaryArchSuffix = $portVersion
+	devel:libclangtidyportabilitymodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyreadabilitymodule$secondaryArchSuffix = $portVersion
+	devel:libclangtidyutils$secondaryArchSuffix = $portVersion
+	devel:libclangtidyzirconmodule$secondaryArchSuffix = $portVersion
+	devel:libclangtooling$secondaryArchSuffix = $portVersion
+	devel:libclangtoolingastdiff$secondaryArchSuffix = $portVersion
+	devel:libclangtoolingcore$secondaryArchSuffix = $portVersion
+	devel:libclangtoolinginclusions$secondaryArchSuffix = $portVersion
+	devel:libclangtoolinginclusionsstdlib$secondaryArchSuffix = $portVersion
+	devel:libclangtoolingrefactoring$secondaryArchSuffix = $portVersion
+	devel:libclangtoolingsyntax$secondaryArchSuffix = $portVersion
+	devel:libclangtransformer$secondaryArchSuffix = $portVersion
+	lib:libclang$secondaryArchSuffix = $portVersionCompat
+	lib:libclang_cpp$secondaryArchSuffix = $portVersionCompat
+	"
+REQUIRES_clang="
+	haiku$secondaryArchSuffix
+	lib:libLLVM$secondaryArchSuffix
```

Comment:
since clang16 it looks like clang defaults to ld.ldd but the dependency is missing